ci-operator/step-registry/openshift-qe/installer/bm/day2/regulus/openshift-qe-installer-bm-day2-regulus-ref.yaml

```diff
@@ -0,0 +1,121 @@
+ref:
+  as: openshift-qe-installer-bm-day2-regulus
+  from_image:
+    namespace: ci
+    name: ocp-qe-perfscale-ci
+    tag: latest
+  cli: latest
+  commands: openshift-qe-installer-bm-day2-regulus-commands.sh
+  credentials:
+    - namespace: test-credentials # was: ci this entry injects the custom credential
+      name: cluster-secrets-metal-perscale-cpt
+      mount_path: /bm
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  env:
+    - name: BAREMETAL               # begin vars for the target cluster
+      default: "true"
+      documentation: |-
+        Set to true if the job is for baremetal
+    - name: KUBECONFIG_PATH 
+      default: "~/mno/kubeconfig"
+      documentation: |-
+        Set kubeconfig path inside the bm, default to ~/mno/kubeconfig
+    - name: LAB_CLOUD   
```

Comment:
This variable is not needed anymore, more context here: https://docs.google.com/presentation/d/1H7EN66MUeygqGk2i-5OfkDEKvP6fX7sKbpvQ9tK-zSo

Reply:
I have removed it.